content/applications/sales/point_of_sale/restaurant/tips.rst

```diff
@@ -2,30 +2,35 @@
 Integrate a tip option into payment
 ===================================
 
-As it is customary to tip in many countries all over the world, it is
-important to have the option in your PoS interface.
-
-Configure Tipping
-=================
-
-To activate the *Tips* feature, go to :menuselection:`Point of Sale
---> Configuration --> Point of sale` and select your PoS.
-
-Under the Bills & Receipts category, you will find *Tips*. Select it
-and create a *Tip Product* such as *Tips* in this case.
-
-.. image:: tips/tips01.png
-    :align: center
-
-Add Tips to the bill
-====================
-
-Once on the payment interface, you now have a new *Tip* button
-
-.. image:: tips/tips02.png
-    :align: center
-
-Add the tip your customer wants to leave and process to the payment.
-
-.. image:: tips/tips03.png
-    :align: center
+Tipping is customary in multiple countries. Point of Sale allows tipping in **shops**, **bars** or
+**restaurants**.
+
+Configuration
+=============
+
+To activate the **Tips** feature, go to :menuselection:`Point of Sale --> Configuration --> Point
+of Sale`, select the POS, scroll down to the :guilabel:`Payment` section and enable
+:guilabel:`Tips`. Once enabled, add a :guilabel:`Tip Product` in the corresponding field, and save.
+The referred product will be used as a reference on customer receipts.
+
+.. image:: tips/tips-setup.png
+   :align: center
+   :alt: enable tips in a POS
+
+.. note::
+   - You can create a product on the spot. To do so, enter your product's name and click
```

Comment:
```suggestion
   - You can create a product on the spot. To do so, enter your product's name and click
```
I guess this is because you have to create a product "tip" to use in the Tip Product field, but this not explained anywhere, nor is it explained how to configure the product in the product template. IMO, this should either be explained or :ref:/:doc: if it is somewhere in the documentation. Also, it should be in the main text and not as a note (perhaps in a H3 heading, as suggested above). If it does come as part of the POS module (as it is available in a runbot under [TIPS] Tips), then it should be documented people can search for that pre-made product (and that it comes by default).

Also, I can see from the screenshot: two options Add tip after payment (NA only), and add tip through payment terminal (Adyen). Neither are explained either, and are only notes (which IMO should be in the main text as well). Also, why only Adyen ? I tried in a runbot and only have Adyen too, if it's only doable in Adyen, it should be documented as it doesn't say otherwise, nor says how to do it otherwise (if possible).

Reply:
You don't have to "create" a product, but if you do, as written, you can just "type the product's name and click create," which creates a product and set it up completely.
However, since it appears that part was unclear, I've added the h3 title about the product creation :)

Only Adyen because it only works for Adyen 
I don't think it should be in the main text; these are just options to activate; there's not enough content for a whole part in the main text; I've let them in the note
However, I added some info about the tip after the payment